spec/after_commit_action_spec.rb

```diff
@@ -18,4 +19,21 @@
     t.array.should include('before_update')
     t.array.should include('after_update')
   end
+
+  context 'when block is executed in another model transaction' do
+    let(:tester)        { Tester.create count: 0 }
+    let(:another_model) { AnotherModel.new tester: tester }
+
+    subject { another_model.save! }
+
+    it { expect { subject }.to change { tester.reload.count }.from(0).to 1 }
+  end
+
+  context 'when there is no transaction' do
```

Comment:
Looking at this some more, why would this run without a transaction? Shouldn't the `create` call automatically trigger a transaction? I see this test fail without the change above, but I can't quite believe this doesn't work as we would have definitely seen this crop up in counter_culture. Can you add some more detail as to why this runs without a transaction?